Form-Controls/styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+  background-image: linear-gradient(
+    to right,
+    rgb(223, 210, 210),
+    rgb(211, 180, 180)
+  );
+  font-family: "EB Garamond", serif;
+}
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+}
+h1 {
+  text-align: center;
+  padding-top: 20px;
+}
+label {
+  font-weight: bold;
+  margin-bottom: 10px;
+  padding-top: 10px;
+}
+h2 {
+  text-align: center;
+}
+input[type="text"],
```

Comment:
Why have you applied this style to the text and select, but not the email input? Can you talk through your decision there?